### DISCUSSION TITLE
Suggestion: Loopy Landscapes scenario change difficulty group

### DISCUSSION THREAD

**Discussion by xrgtmx** [OpenRCT2/OpenRCT2#21377] [Feature requests]
Can Southern Sands, Tiny Towers, Nevermore Park and Pacifica be moved from Challenging to Expert parks?

**Comment by ZeeMaji** [comment]
Seems sensible to me, these parks seem a lot more expert leaning than late challenging. The categorization they currently have is a strictly OpenRCT2 thing as vanilla RCT1 did not have difficulty markers in the scenario list.

**Comment by xrgtmx** [comment]
Hello, checking in to see if this change can be implemented?
